Jenkinsfile

```diff
@@ -65,31 +73,35 @@ pipeline {
                         }
                     }
 
-                    // env.PR_GITHUB_TOKEN = sh (returnStdout: true, script : 'oc get secret labs-robot-github-oauth-token --template=\'{{.data.password}}\' | base64 -d')
-                    env.PR_GITHUB_TOKEN = new String("oc get secret labs-robot-github-oauth-token --template='{{.data.password}}'".execute().text.minus("'").minus("'").decodeBase64())
                     env.PR_CI_CD_PROJECT_NAME = "labs-ci-cd-pr-${env.PR_ID}"
                     env.PR_DEV_PROJECT_NAME = "labs-dev-pr-${env.PR_ID}"
                     env.PR_TEST_PROJECT_NAME = "labs-test-pr-${env.PR_ID}"
+
+                    env.PR_GITHUB_TOKEN = new String("oc get secret labs-robot-github-oauth-token --template='{{.data.password}}'".execute().text.minus("'").minus("'").decodeBase64())
                     if (env.PR_GITHUB_TOKEN == null || env.PR_GITHUB_TOKEN == ""){
                         error('PR_GITHUB_TOKEN cannot be null or empty')
                     }
                     env.USER_PASS = "${env.PR_GITHUB_USERNAME}:${env.PR_GITHUB_TOKEN}"
+
+                    env.PR_BRANCH = "cleanup"
+//                    env.PR_BRANCH = "pull/${env.PR_ID}/head"
```

Comment:
Looks like a "left-over" from some testing? Probably should be reverted(?)

Reply:
No reason for the name. I can make it whatever.

Correct, that is there because it needed to be on that branch to test. It
would be reverted back to the line below just before merging. It's that
whole thing about testing a Jenkinsfile on a PR before it's in master

Reply:
i could also change this behavior so that instead of using the `pull/${env.PR_ID}/head` it would use the actual branch name. but i'm not keen on that because then you have to go to the fork instead of just using this repo. It's def one of the benefits of GitHub, it just makes your PR into a branch on the original repo instead of having to go to the fork.

Reply:
```
  PIPELINE_SOURCE_REPOSITORY_REF: cleanup
  PIPELINE_SOURCE_REPOSITORY_URL: https://github.com/pcarney8/labs-ci-cd.git
```
^ in ci-for-labs-ci-cd.yml will also need to be changed before merge as well. i'd like to use a tag instead of `master` for the ref